### PR TITLE
Add Flint queries for Apache Access integration

### PR DESCRIPTION
**Pull request by Swiddis**
### Description
Adds some queries with simple mappings. Future work needed for more complicated fields (like user agent parsing).

### Issues Resolved
N/A

### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass, including unit test, integration test and doctest
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
  - [ ] New functionality has user manual doc added
- [X] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by YANG-DB — CHANGES_REQUESTED** [review]
Can you please update according with the next list guidelines
opensearch-project/opensearch-catalog#144

File: server/adaptors/integrations/__data__/repository/apache/assets/create_table-1.0.0.sql [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  empty_col STRING,
+  remote_user STRING,
+  time_local_1 STRING,
+  time_local_2 STRING,
+  request STRING,
+  status INT,
+  body_bytes_sent INT,
+  http_referer STRING,
+  http_user_agent STRING
+) USING csv
```
[the remark this answers is not in the document]
Comment (Swiddis) [reply server/adaptors/integrations/__data__/repository/apache/assets/create_table-1.0.0.sql:12]: We can incrementally add formats, so it shouldn't be a blocker to add one now and add a parquet workflow in a second PR.

**Comment by Swiddis** [comment]
> Can you please update according with the next list guidelines [opensearch-project/opensearch-catalog#144](https://github.com/opensearch-project/opensearch-catalog/issues/144)

Can you make this feedback more actionable? That's a large RFC and I'm not certain which parts I'm meant to apply right now.

**Review by RyanL1997 — APPROVED** [review]
LGTM. Just awaiting on addressing @YANG-DB's comments.

**Comment by YANG-DB** [comment]
Hi
Table name 
MV query name 
Dashboard name 
Index pattern name 
Visualizations name

**Comment by Swiddis** [comment]
Commented on the RFC, I don't think these changes are applicable to 2.13, and are at least out of scope for this PR.

**Review by YANG-DB — APPROVED** [review]

**Review by YANG-DB — APPROVED** [review]

**Review by ps48 — APPROVED** [review]

**Review by paulstn — APPROVED** [review]

**Pull request merged**